src/eth/common.rs

```diff
@@ -118,10 +118,10 @@ impl AbiType for U256 {
 	const IS_FIXED: bool = true;
 }
 
-impl AbiType for Address {
+impl AbiType for H160 {
```

Comment:
maybe it would be better to declare `type Address` and export it as `pwasm_abi::types::Address`.. Which also will remove the need to declare it [here](https://github.com/paritytech/pwasm-abi/pull/77/files#diff-11de442f8c13daab760fd46822928e3dR178).

Reply:
That is what is going to happen when `pwasm-std` version `0.12` is going to be released.